packages/react/src/hooks/useConnect.tsx

```diff
@@ -94,6 +99,14 @@ export const useConnect = (props?: Props) => {
       );
       return;
     }
+
+    trackCustomEvent({
+      name: EVT_TREASURECONNECT_UI_LOGIN,
+      properties: {
+        isUsingTreasureLauncher,
```

Comment:
This will always be `false` in its current placement. If that's desired, that's fine

Reply:
Yeah I think I'll just remove it